src/python/pygrackle/yt_fields.py

```diff
@@ -71,7 +71,7 @@
     'x-velocity': (('gas', 'velocity_x'), 'code_velocity'),
     'y-velocity': (('gas', 'velocity_y'), 'code_velocity'),
     'z-velocity': (('gas', 'velocity_z'), 'code_velocity'),
-    'energy': (('gas', 'thermal_energy'), 'code_velocity**2'),
+    'energy': (('gas', 'specific_thermal_energy'), 'code_velocity**2'),
```

Comment:
Need to revert until using yt-4.0.
```suggestion
    'energy': (('gas', 'thermal_energy'), 'code_velocity**2'),
```